### PR TITLE
Rewritten ldap_status.sh

### PR DESCRIPTION
**Pull request by vyskocilpavel**
* Using variable names in uppercase
* Add time counter

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -82,20 +82,20 @@ This script checks if the LDAP servers are accessible
...
 
     # LDAP password
-    password=""
+    PASSWORD=""
 
     # Base dn of LDAP tree
-    basedn=""
+    BASEDN=""
 
     # eduPersonPrincipalName which will be searched
```
Comment (dBucik) [inline README.md]: ... which the script will look for

File: README.md [hunk 2]
```diff
@@ -82,20 +82,20 @@ This script checks if the LDAP servers are accessible
...
-    searchedIdentity=""
+    IDENTITY=""
 
-    # List of LDPA hostnames separated by space
+    # List of LDPA HOSTNAMES separated by space
```
Comment (dBucik) [inline README.md]: typo in LDPA --> LDAP

Comment (dBucik) [reply README.md]: I would also replace "space" with "whitespace"

File: ldap_status.sh [hunk 3]
```diff
@@ -1,32 +1,35 @@
...
-basedn=""
+BASEDN=""
 
 # eduPersonPrincipalName which will be searched
-searchedIdentity=""
+IDENTITY=""
 
-# List of LDPA hostnames separated by space
+# List of LDPA HOSTNAMES separated by space
 # Included ldap:// or ldaps://
```
Comment (dBucik) [inline ldap_status.sh]: I think this wording is a bit confusing. I.e., it seems a bit unclear to me if I should add the ldaps:// or not. Perhaps change the text as well?

**Review by dBucik — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (vyskocilpavel) [reply README.md]: Resolved

File: ldap_status.sh (hunk 3 above)
Comment (vyskocilpavel) [reply ldap_status.sh]: Rewrited.

**Review by dBucik — APPROVED** [review]

**Pull request merged**